### ISSUE TITLE
How do we use the validation sets in code？

### ISSUE THREAD

**Issue by wangsen-zy** [layumi/Person_reID_baseline_pytorch#180]
I found that the validation set is taken from the training set，I think the purpose of the validation set may be to adjust the hyperparameters.
①：How did we do that in codes?
②：Why does the training and validation sets overlap,will this cause overfitting？

**Comment by wangsen-zy** [comment]
I just found some answers in your other comments，knowing that your validation set overlapping the training set is set for security reasons.But I still confused about how to code the validation set to debug hyperparameters?Hope to get some feedback from you and thanks in advance!

**Comment by layumi** [comment]
@wangsen-zy 
1. First of all, The original Market-1501 do not have a validation set. We need to split the dataset by ourself.
2. There are two ways to split the original training set into the training set and validation set.
- classification split: just randomly set some images as the validation set. (which is in this code. Because it is easy to implement.)
- retrieval split: I set 100 classes as validation, and only use the 651 classes to training. (what I do when I submit my paper.)

**Comment by wangsen-zy** [comment]
Thank you very much for your reply, this solved my confusion, and all the best to you.
